src/main/java/seedu/address/logic/Messages.java

```diff
@@ -1,4 +1,4 @@
-package seedu.address.commons.core;
```

Comment:
```Furthermore, in the next few commits we will be adding methods to the
Messages class which require access to model classes. This would not be
possible if the Messages class is still in the commons layer, which is
the lowest layer of our architecture and can't access component.
```

This above commit message suggests that this commit should be part of the toString PR.
In future, let's keep commits of those AB4 PRs together unless the they don't apply to AB3. There may be good reasons for the commits to be together and in that order, although we might not catch them at first.

For now, we have two options:
1. Merge this PR first and the toString PR immediately after.
2. Add this commit to the toString PR and close this one.